### PR TITLE
Updated module versions and example configuration for Automanage Machine Configuration custom configurations for Linux

### PR DESCRIPTION
**Pull request by janegilring**
- Updated PowerShell module versions
- Added multiple DSC resources to the example configuration since the https://github.com/Azure/nxtools/issues/15 is now resolved

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ While the use of custom configurations in Automanage Machine Configuration featu
 
 By the end of this scenario, you will have both Linux and Windows Azure Arc-enabled servers with Automanage Machine Configuration custom configurations assigned.
 
-This scenario starts at the point where you already deployed **[Jumpstart ArcBox for IT Pros](/azure_jumpstart_arcbox/itpro/)** and have 5 Azure Arc-enabled servers in the resource group is deployed to visible as resources in Azure.
+This scenario starts at the point where you already deployed **[Jumpstart ArcBox for IT Pros](https://azurearcjumpstart.io/azure_jumpstart_arcbox/itpro/)** and have 5 Azure Arc-enabled servers in the resource group is deployed to visible as resources in Azure.
 
-![Screenshot of Azure portal showing Azure Arc-enabled servers](./01.png)
+![Screenshot of Azure Portal showing Azure Arc-enabled servers](./01.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md:19]: Accessibility review team asked lowercase

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ The nxtools module contains DSC resources used for the demo configuration.
 
 ## Azure resources
 
-> **Note:** For the remaining code blocks in this article, copy the code into ```/home/arcdemo/MachineConfiguration.ps1``` , mark the lines you want to run and click F8. Alternatively, right click the selected commands and click Run Selection.
+> **NOTE: For the remaining code blocks in this article, copy the code into ```/home/arcdemo/MachineConfiguration.ps1``` , mark the lines you want to run and click F8. Alternatively, right click the selected commands and click Run Selection:**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md:133]: Keep the new format

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md [hunk 3]
```diff
@@ -311,34 +334,34 @@ if ($roleDefinitionIds.Count -gt 0)
...
 
 It may take 15-20 minutes for the policy remediation to be completed.
 
 Get a Machine Configuration specific view by following these steps:
 
-- In the Azure portal, navigate to *Azure Arc* -> *Servers*
-- Click on ArcBox-Ubuntu-01 -> Machine Configuration
-- If the status for *ArcBox-Ubuntu-01/AzureArcJumpstart_Linux* is not *Compliant*, wait a few more minutes and click *Refresh*
+- In the Azure Portal, navigate to *Azure Arc* -> *Servers*
+- Click on Arcbox-Ubuntu-01 -> Machine Configuration
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md:348]: this is not ArcBox

**Review by likamrat — CHANGES_REQUESTED** [review]
See comments

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md (hunk 1 above)
Comment (janegilring) [reply docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md]: Updated PR

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md (hunk 2 above)
Comment (janegilring) [reply docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md:133]: Updated PR

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md (hunk 3 above)
Comment (janegilring) [reply docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md:348]: The scenario is using ArcBox as a prerequisite:
https://github.com/Azure/arc_jumpstart_docs/blob/automanage_machine_configuration_updates/docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md?plain=1#L17

**Pull request closed without merging**